.eslintrc.json

```diff
@@ -1,14 +1,15 @@
 {
-  "parser": "babel-eslint",
+  "parser": "vue-eslint-parser",
```

Comment:
I'm cautious here: so `vue-eslint-parser` wraps `babel-eslint`, and basically preparses Vue.js files and relies on `babel-eslint` to parse non Vue.js-specific input, right? I.e. is the flow:

```text
{ Vue.js Input }
    |
    v
[[ vue-eslint-parser ]]
    |                          \
    v                           v
[[ vue-parser ]]        [[ babel-eslint ]] ==========> {JavaScript AST}
    |                            ^
    v                         /
[[ Transpiled JavaScript ]]
```

Reply:
I haven't look into it... But I guess it should be?